backward_right_actuator:
  actuator: 'EPOS2'
  protocol: 'MAXON SERIAL V2'
  interface: 'USB'
  serial_number: '602079024057'
  is_sub_device: false
  node_id: 1
  operation_mode: 'profile_velocity'
  clear_fault: true

  position_profile:
    velocity: 589       # [rpm]
    acceleration: 1178 # [rpm/s]
    deceleration: 1178 # [rpm/s]
  velocity_profile:
    acceleration: 1178 # [rpm/s]
    deceleration: 1178 # [rpm/s]

forward_right_actuator:
  actuator: 'EPOS2'
  serial_number: '602079023956'
  protocol: 'CANopen'
  is_sub_device: true
  master_device: 'backward_right_actuator'
  node_id: 3
  operation_mode: 'profile_velocity'
  clear_fault: true

  position_profile:
    velocity: 589       # [rpm]
    acceleration: 1178 # [rpm/s]
    deceleration: 1178 # [rpm/s]
  velocity_profile:
    acceleration: 1178 # [rpm/s]
    deceleration: 1178 # [rpm/s]

backward_left_actuator:
  actuator: 'EPOS2'
  serial_number: '602079023838'
  protocol: 'CANopen'
  is_sub_device: true
  master_device: 'backward_right_actuator'
  node_id: 2
  operation_mode: 'profile_velocity'
  clear_fault: true

  position_profile:
    velocity: 589       # [rpm]
    acceleration: 1178 # [rpm/s]
    deceleration: 1178 # [rpm/s]
  velocity_profile:
    acceleration: 1178 # [rpm/s]
    deceleration: 1178 # [rpm/s]

forward_left_actuator:
  actuator: 'EPOS2'
  serial_number: '602079024066'
  protocol: 'CANopen'
  is_sub_device: true
  master_device: 'backward_right_actuator'
  node_id: 4
  operation_mode: 'profile_velocity'
  clear_fault: true

  position_profile:
    velocity: 589       # [rpm]
    acceleration: 1178 # [rpm/s]
    deceleration: 1178 # [rpm/s]
  velocity_profile:
    acceleration: 1178 # [rpm/s]
    deceleration: 1178 # [rpm/s]